jwst/resample/resample.py

```diff
@@ -36,7 +38,7 @@ class ResampleData:
     drizpars = {'single': False,
                 'kernel': 'square',
                 'pixfrac': 1.0,
-                'good_bits': 4,
+                'good_bits': CRBIT,
```

Comment:
Remind me why it is we want resample to treat pixels flagged as CR as a good pixel (instead of rejecting them)?

Reply:
No one else has provided any other value to use as a default.  This can certainly be updated to better match what the instrument teams feel would be best.  We just need a new value from someone...

Reply:
Since this is being run after outlier_detection, which inserts JUMP_DET flags, we definitely don't want to treat those as good. Looking over the list of predefined flag conditions I don't really see any that would be totally harmless or OK to include when combining data, so I would set the default to zero (i.e. reject anything with DQ > 0). That way we also automatically get rid of reference pixels from mosaics, assuming of course that the reference pixels were flagged upstream with `REFERENCE_PIXEL`.